dialog/file.go

```diff
@@ -20,13 +20,14 @@ type textWidget interface {
 }
 
 type fileDialog struct {
-	file       *FileDialog
-	fileName   textWidget
-	dismiss    *widget.Button
-	open       *widget.Button
-	breadcrumb *widget.Box
-	files      *fyne.Container
-	fileScroll *widget.ScrollContainer
+	file            *FileDialog
+	fileName        textWidget
+	dismiss         *widget.Button
+	open            *widget.Button
+	breadcrumb      *widget.Box
+	files           *fyne.Container
+	fileScroll      *widget.ScrollContainer
+	showHiddenCheck *widget.Check
```

Comment:
We generally avoid putting types in variable names, Go likes to avoid duplication